### ISSUE TITLE
hbook: missing h2d binning accessor

### ISSUE THREAD

**Issue by mughetto** [go-hep/hep#70]
Unless I've missed it, it's impossible to loop on H2 bins like we can for H1s:
```
func tester(sig *hbook.H1D, bkg *hbook.H1D) {
        for i, bin := range sig.Binning().Bins() {
                fmt.Println(i,bin)                              
        }
}
```

A simple accessor like below seems missing for h2d:
https://github.com/go-hep/hep/blob/master/hbook/h1d.go#L89

**Comment by sbinet** [comment]
yep, seems like so.

want to give it a try?

**Comment by mughetto** [comment]
I'm on it

**Comment by sbinet** [comment]
Closed via go-hep/hep#71
